content/blog/18-19/chat2.md

```diff
@@ -0,0 +1,76 @@
+
+# Hoe stap je over?
+
+1. Ga naar de mattermost server <https://mattermost.zeus.gent> of <https://chat.zeus.gent>
+    - met je favoriete browser
+    - met de [Android-app](https://play.google.com/store/apps/details?id=com.mattermost.rn)
+    - de [iOS-app](https://itunes.apple.com/us/app/mattermost/id1257222717?mt=8).
+2. Maak een account aan met een email naar keuze, ook niet-ugent emails worden geaccepteerd. De accounts zijn ook (nog) niet gekoppeld aan je Zeus-account.
+3. Join je favoriete kanalen (~cats en ~top-secret-stuff zijn aanraders)
+
+## De belangrijkste veranderingen ten opzichte van Slack
+
+Eerst en vooral: nu de Mattermostserver zelf gehost wordt hebben de sysadmins volledige toegang tot de databank. Dit wil zeggen dat zij al jullie berichten (ook direct-messages) kunnen lezen, moesten ze dat willen. Niet dat ze dat ook gaan doen. Maar als je wilt confidentiële gesprekken hebben dan gebruik je hier best een applicatie voor met end-to-end encryptie (bijvoorbeeld [Signal](https://www.signal.org/)). Bij Slack kunnen de werknemers daar al je berichten kijken, dus eigenlijk verandert er niet veel.
+
+- Je refereert naar kanalen met een ~ i.p.v. een #.
+- Tab-completion werkt niet zonder @.
+- De (niet web)-clients ondersteunen geen meerdere servers, als je dus meerdere workspaces hebt, moet je helaas gebruik maken van iets zoals een app cloner, voor Android werkt [deze](https://play.google.com/store/apps/details?id=com.applisto.appcloner&hl=en) alvast.
+- De naam van een kanaal veranderen verandert alleen de display name, in de URL blijft de originele naam.
+- \#zeus wordt ~zeus, \#general wordt ~off-topic, maar die dingen lijken nog wat te moeten settelen aangezien er wierig aan naamsverandering wordt gedaan voorlopig.
+- Je krijgt mails over mentions, maar die kan je uitzetten.
+- `/door` werkt, al kan het wel zijn dat je eerst 'regular-user' status moet krijgen, poke een admin als je die rechten wilt.
+- `/cammiechat`, en `/stock` werken ook
+- De integraties vind je op <https://github.com/ZeusWPI/mattermore>, iedereen wordt aangemoedigd om nieuwe dingen toe te voegen.
+- Sommige andere integraties werken nog niet (quotes bijvoorbeeld), maar daar wordt aan gewerkt.
+- Het bestuur-kanaal zal integraal naar Mattermost verhuizen (naar ~bestuur), bestuurszaken gaan we vanaf nu dus bespreken daar en niet meer op Slack.
+- Custom emoji zijn nog steeds een ding 🎉.
+
+## Wat staat er nog op de planning?
+
+- We gaan misschien de Mattermost app zelf builden en publiceren op de appstores zodat de clone hack niet nodig is.
+- We gaan wat experimenteren met bridges (naar Slack, IRC, ...) om op zoveel mogelijk plaatsen aanspreekbaar te zijn.
+- Dit wordt allemaal nog eens via mail gecommuniceerd ook.
+- Veel custom responses en gifs zijn nog niet er nog niet _!wenkbrauw_.
+
```

Comment:
We wilden misschien ook zelf de mattermost repo clonen zodat we gemakkelijker zelf patches aan de code kunnen toevoegen, misschien hier ook vermelden?

Reply:
Zoals besproken op de vergadering: een eigen fork vergt veel werk om bij te houden, we gaan beginnen met PR's te sturen naar de Mattermost-repo en eventueel maken we onze eigen fork als dat regelmatig gebeurt.